### PR TITLE
f-checkout@0.30.0 - Update readme

### PR DESCRIPTION
**Pull request by FayeCarter**
### Changed
- props, CSS class, and event descriptions added to `README`.

### Removed
- unused event names.

### CODE REVIEW

File: packages/f-checkout/README.md [hunk 1]
```diff
@@ -55,7 +55,85 @@
...
+
+### Props
+
+`f-checkout` has a number of props that allow you to customise its functionality.
+
+The props that can be defined are as follows:
+
+| Prop  | Type  | Default | Description |
+| ----- | ----- | ------- | ----------- |
+| `checkoutUrl` | `String` | - | Sets the url for the API called to load the Checkout Data.<br><br>The data returned from this API contains the serviceType, which determines if the Checkout component is created for Collection or Delivery. |
```
Comment (SachaZvetelman) [inline packages/f-checkout/README.md:69]: We're missing the recently introduced URL for available fulfilment times.

File: packages/f-checkout/README.md [hunk 2]
```diff
@@ -55,7 +55,85 @@
...
+| `checkoutTimeout` | `Number` | `1000` | Sets the timeout when submitting the checkout form. |
```
Comment (DamianMullins) [inline packages/f-checkout/README.md:71]: Looks like you're missing `checkoutAvailableFulfilmentUrl`?

File: packages/f-checkout/README.md [hunk 3]
```diff
@@ -55,7 +55,85 @@
...
+| `c-userNote-title` | Can be used to target the checkout user note title. |
+| `c-userNote-content` | Can be used to target the checkout user note text description. |
+| `c-userNote-textArea` | Can be used to target the checkout user note text area. |
+
+### Events
+
+| Event | Description |
+| ----- | ----------- |
+| `checkout-payment-success` | This event is emitted when checkout form is sucessfully submitted. |
+| `checkout-payment-failure` | This event is emitted when checkout form is fails when submitted. |
```
Comment (SachaZvetelman) [inline packages/f-checkout/README.md]: Typo here 😄

File: packages/f-checkout/README.md [hunk 4]
```diff
@@ -55,7 +55,85 @@
...
+| `checkout-get-success` | This event is emitted when checkout data is sucessfully loaded.  |
+| `checkout-get-failure` | This event is emitted when checkout data is fails to load. |
```
Comment (SachaZvetelman) [inline packages/f-checkout/README.md]: Same here.

File: packages/f-checkout/README.md [hunk 5]
```diff
@@ -55,7 +55,85 @@
...
+
+You can add event listeners for these like so
+
+```js
```
Comment (DamianMullins) [inline packages/f-checkout/README.md]: `html` might be a better choice of language here as it'll handle the syntax highlighting a little better.

File: packages/f-checkout/README.md (hunk 4 above)
Comment (FayeCarter) [reply packages/f-checkout/README.md]: 🤦‍♀️

File: packages/f-checkout/README.md (hunk 3 above)
Comment (FayeCarter) [reply packages/f-checkout/README.md]: 🤦‍♀️

File: packages/f-checkout/README.md (hunk 2 above)
Comment (FayeCarter) [reply packages/f-checkout/README.md:71]: Ahh well spotted, thanks 🙂

File: packages/f-checkout/README.md (hunk 1 above)
Comment (FayeCarter) [reply packages/f-checkout/README.md:69]: Ahh well spotted, thanks 🙂

**Review by DamianMullins — APPROVED** [review]

**Review by SachaZvetelman — APPROVED** [review]

File: packages/f-checkout/README.md [hunk 6]
```diff
@@ -56,6 +56,86 @@
...
+| `checkoutAvailableFulfilmentUrl` | `String` | - | Sets the url for the API called to load the Available Fulfilment data. |
+| `checkoutTimeout` | `Number` | `1000` | Sets the timeout when submitting the checkout form. |
+| `getCheckoutTimeout` | `Number` | `1000` | Sets the timeout when loading checkout data. |
+| `authToken` | `String` | `''` | Sets the authorisation token used when submitting the checkout form. |
+
+### CSS Classes
+
+Checkout has its own styles which are scoped to the component using CSS modules to prevent conflicts with existing styles on the page.
+
+In addition to this, checkout exposes some classes which you can target in your application.
```
Comment (ashleynolan) [inline packages/f-checkout/README.md:79]: I don't think this is true in checkout (it is in the modal component). I'd remove this line, as all checkout styles are scoped to the component.

Up to you if you want to leave this CSS classes section in – think it's probably only needed where the class names are possible to hook onto in a consuming application.

**Pull request merged**